saleor/account/utils.py

```diff
@@ -101,3 +104,20 @@ def get_random_avatar():
     avatar_name = random.choice(os.listdir(AVATARS_PATH))
     avatar_path = os.path.join(AVATARS_PATH, avatar_name)
     return File(open(avatar_path, "rb"), name=avatar_name)
+
+
+def remove_staff_member(staff):
```

Comment:
Shouldn't we separate staff actions from normal users? I mean, maybe we should have another file for the logic which comes from dashboard 1.0?

Reply:
It seems that everything fit somewhere except for this staff functions (and one of them: `staff_member_required` will be removed soon). So we can create separate file for this stuff actions, but it will be for one function. Also user and staff user have a lot in common (only `is_staff` field differ them), actions applicable to user are also applicable to staff user, so I am not sure if this distinction is needed.